pootle/apps/pootle_app/models/directory.py

```diff
@@ -57,7 +57,7 @@ class Meta(object):
         ordering = ['name']
         app_label = "pootle_app"
 
-    ############################ Properties ###################################
+    # # # # # # # # # # # # # #  Properties # # # # # # # # # # # # # # # # # #
```

Comment:
I suppose that using other character doesn't fix the problem entirely because other rule would enforce a blank space to be after the initial hash, isn't it?